src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
```

Comment:
mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-use-static-methods-in-your-solution

Reply:
I know about the static methods in Lottery and ColorSupplier and that this is a bad practice in general, but honestly, I think it should be there because we are not manipulating any data of the objects with those methods and using them only to get randoms, so there is no need to create objects of those classes that I mentioned above.

Reply:
I can easily change this to non-static, but still wanted to clarify

Reply:
Let's stick to the suggested guidelines and if you want to go deeper on why is it a bad practice to use static methods you can ask this question in tomorrow's lesson.

Reply:
I believe Bohdan will be able to answer it better than I can.

Reply:
> Let's stick to the suggested guidelines and if you want to go deeper on why is it a bad practice to use static methods you can ask this question in tomorrow's lesson.

ok, no problems.